incubator/hnc/test/e2e/demo_test.go

```diff
@@ -183,9 +183,9 @@ spec:
 		MustRun("kubectl hns create", nsService3, "-n", nsTeamA)
 		expected := "" + // empty string make go fmt happy
 			nsTeamA + "\n" +
-			"├── " + nsService1 + "\n" +
-			"├── " + nsService2 + "\n" +
-			"└── " + nsService3
+			"├── [s] " + nsService1 + "\n" +
+			"├── [s] " + nsService2 + "\n" +
+			"└── [s] " + nsService3
```

Comment:
Oh, this change should be merged in the same kubeclt-hns plugin change, right?

Was this the main cause of the failed e2e test at HEAD?

Reply:
It's one of the causes but there seem to be many others too.